README.md

```diff
@@ -62,7 +82,12 @@ victor {
     generateVectorDrawables = true
 }
 ```
-
+and in Kotlin
+```kotlin
+configure<com.trello.victor.VictorPluginExtension> {
+    generateVectorDrawables = true
+}
+```
```

Comment:
I like providing these docs, but could we split them out into its own `# Usage (Kotlin Gradle)` section? I worry that interleaving these examples between two different types of Gradle files will be confusing.